pages/library/_viewId.vue

```diff
@@ -54,6 +52,20 @@ export default Vue.extend({
 
       this.items = itemsResponse.data.Items || [];
     }
+  },
+  methods: {
+    imageLink(id: string) {
```

Comment:
This was moved to the component so it shouldn't be necessary, afaik.

Reply:
Yup, not needed. I forgot to remove it :p